LICENSE

```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2020, Power Xpert Blue
+Copyright (c) 2020 - present, Eaton
```

Comment:
Can you change this to say Brightlayer UI

Reply:
@daileytj , in other repositories  we are using `Copyright (c) 2020 - present, Eaton` instead of `Copyright (c) 2020, Power Xpert Blue`
Please check for reference [Repo URL](https://github.com/pxblue/angular-workflows/blob/dev/LICENSE#L3)

Reply:
I'm out of the loop on that one. Thanks for pointing that out... Sounds good to me.